src/hdf5/osw_2ioda.py

```diff
@@ -318,19 +338,23 @@ def quality_control(obs_data):
                           help='base date for the center of the window')
     optional.add_argument('--debug', action='store_true',
                           help='enable debug messages')
-    optional.add_argument('--verbose', action='store_true',
-                          help='enable verbose debug messages')
+    optional.add_argument('-q', '--qualitycontrol', dest='qc_strict',
+                          default=False, action='store_true', required=False,
+                          help='turn on quality control georeality checks')
```

Comment:
Not sure what georeality check is, Is this more of a PreQC? It's a little confusing given there there is `disableQC` in Simobs after the simulation.

Reply:
this is similar to other data converter... 

when this option is present in the converter is applies strict QC to the data dropping the *bad* data points

the `disableQC` in the SIMOBS is hardcoded to look for `EffectiveQC` (not `PreQC`) which is what is assigned through Skylab (and generally assimilation)